src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
+  const adder = (numbers) => {
+    if (numbers === undefined) {
+      const result = count;
+
+      count = 0;
+
+      return result;
+    } else {
+      count += numbers;
+
+      return adder;
+    }
```

Comment:
you don't need else here, anyways you return value from previous if

```suggestion
    }
   
    count += numbers;

    return adder;
```